stl/inc/deque

```diff
@@ -43,8 +41,9 @@ public:
         : _Mycont(static_cast<const _Mydeque*>(_Pdeque)), _Myoff(_Off) {}
 
     _NODISCARD reference operator*() const noexcept {
-        _Size_type _Block = _Mycont->_Getblock(_Myoff);
-        _Size_type _Off   = _Myoff % _Block_size;
+        constexpr int _Block_size = _Mydeque::_Get_block_size();
+        _Size_type _Block         = _Mycont->_Getblock(_Myoff);

```

Comment:
While you are there both could be `const`

Reply:
I made index variables `const` when possible. But I'm not sure if there are some other variables expected to be made `const`.